radon/tests/test_raw.py

```diff
@@ -246,7 +246,7 @@ def foo(n=1):
                 This is a string not a comment
                 """
         test = 0
-    ''', (6, 6, 9, 1, 3, 0, 0)),
+    ''', (9, 5, 6, 1, 3, 0, 0)),
```

Comment:
I think that LOC shouldn't be concerned with backslash continuation, but it should count all the lines it see. In this case it should be 10, don't you agree?